client/src/views/Home.vue

```diff
@@ -80,15 +81,13 @@ export default {
     },
   },
   created() {
-    this.location_ = getLocationFromRoute(this.$route);
-    this.setLocation(this.location_);
+    this.setLocation(getLocationFromRoute(this.$route));
```

Comment:
I think ideally one would use https://github.com/vuejs/vuex-router-sync rather than trying to remember to `setLocation` in every view that may get loaded